lib/OrdinaryDiffEqDefault/src/composite_algs.jl

```diff
@@ -1,6 +1,7 @@
 ### AutoSwitch
 ### Designed to switch between two solvers, stiff and non-stiff
 
+using OrdinaryDiffEq: Vern6, Vern7, Vern8, Vern9
```

Comment:
Autoswitch itself should live in the core package. 73-79 should live with the methods that use.  The default package only needs 83 onwards.